samples-test/samples/HTML/site.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Centered Button and Label</title>
+    <style>
+        .container {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+            height: 100vh;
+            gap: 10px;
+        }
+        .label {
+            font-family: Arial, sans-serif;
+            font-size: 18px;
+        }
+        .button {
+            padding: 10px 20px;
+            border-radius: 25px;
+            border: none;
+            background-color: #4CAF50;
+            color: white;
+            font-size: 16px;
+            cursor: pointer;
+        }
+        .button:hover {
+            background-color: #45a049;
+        }
+    </style>

```

Comment:
Currently, this HTML is more CSS than "plain" HTML. If I implement a classifier that analyzes the samples, it might think that HTML looks like CSS. So, while it's great that this sample has a `<style>` tag, it will need more HTML so that any future classifier can be more accurate.